### PR TITLE
fix: add aria label to select element for content types

### PR DESCRIPTION
**Pull request by ediiotero**
Added an aria-label to the content type selector to improve accessibility

Motivation and Context
Imporve swagger-ui accessibility

How Has This Been Tested?
Tested with aXe and a screen reader

### CODE REVIEW

File: src/core/components/content-type.jsx [hunk 1]
```diff
@@ -47,7 +47,7 @@ export default class ContentType extends React.Component {
 
     return (
       <div className={ "content-type-wrapper " + ( className || "" ) }>
-        <select className="content-type" value={value || ""} onChange={this.onChangeWrapper} >
+        <select className="content-type" aria-label="Select content type" value={value || ""} onChange={this.onChangeWrapper} >
```
Comment (alexandervalencia) [inline src/core/components/content-type.jsx:50]: non-blocking: my only thought on this is, looking through the places that `ContentType` is used, it seems that they already have labels or label-like elements in non-label tags, like `<span>`:

- ParamBody - https://github.com/department-of-veterans-affairs/swagger-ui/blob/master/src/core/components/param-body.jsx#L134-L135
- Parameters - https://github.com/department-of-veterans-affairs/swagger-ui/blob/master/src/core/components/parameters/parameters.jsx#L216
- Response - https://github.com/department-of-veterans-affairs/swagger-ui/blob/master/src/core/components/response.jsx#L187-L189
- Responses - https://github.com/department-of-veterans-affairs/swagger-ui/blob/master/src/core/components/responses.jsx#L94

Maybe this is a refactor for further down the line, but it seems like we should have a consistent way this select is being labeled using the label description of its given context. Right now, some components have an empty label field, some are using `<span>`s... if we hard-code the aria-label value here we end up with some strange conflicts, like the response control having the "label": "Media type", but `aria-label`: "Select content type".

**Review by alexandervalencia — COMMENTED** [review]
I had a thought that I don't want to hold up this PR but I think we is worth considering moving forward

File: src/core/components/content-type.jsx (hunk 1 above)
Comment (mjlumetta) [reply src/core/components/content-type.jsx:50]: yeah, you know what, this is a really good callout. I noticed the "Media type" element earlier today when I was looking at something else and had a similar thought; didn't know about the `<label>` elements. thinking we should maybe make the consuming component responsible for the label, possibly by exposing `aria-label` and/or `aria-labelledby` props? regardless, in the interest of getting it right, I do think this is probably a blocking question.

**Comment by mjlumetta** [comment]
@gileswells @alexandervalencia @ndsprinkle do we want to hold off on final reviews while Eddie is out? Alex's comment + needing to run `npm run build` seem like issues that need to be resolved, and I think it's fine to wait for Eddie to discuss since this is his ticket. thoughts?

**Comment by gileswells** [comment]
Rebased off `origin/upstream-main` as there were conflicts with the built files. When that broke everything I recreated @ediiotero's change and force pushed that up to this branch so it's mergeable. (Sorry Eddie)

From what I've seen in PRs to `upstream/main` there aren't changes pushed to the built files with a regular PR. It seems like those re reserved for when they're building out new releases.

**Comment by gileswells** [comment]
@ediiotero can you remove the two dist files from your PR. From what I've seen on the upstream repo they don't include those in their PRs despite not having them in the gitignore file.

**Review by gileswells — APPROVED** [review]
LGTM. I just added removed the changes to the dist folder to match the PR style on upstream of not including those on PRs.

**Review by alexandervalencia — APPROVED** [review]
There's a weird thing here where some of the places using `ContentType `are wrapped in `<label>`s... and in those cases it would be better to use the correct HTML relationships (having an `id` on the input and referencing the input on the label with the `for` attribute)... but because some uses aren't associated with labels that wouldn't work... 

For the time being, I think this is an acceptable fix for now

**Review by ndsprinkle — APPROVED** [review]

**Pull request merged**